nodeBasics/index.js

```diff
@@ -0,0 +1,14 @@
+import chalk from 'chalk';
+
+const Temperature = -1;
+const Pressure = 749;
+const Humidity = 91;
+const Wind = 3.0;
+const Description = "From morning to evening, the sky in Kyiv is covered with clouds. It snowed lightly throughout the day, which ended only closer to the evening.";
```

Comment:
on the next lesson we will learn what is wrong with naming here. But right now I do not count it is as mistake